src/_locales/en/messages.json

```diff
@@ -150,8 +150,13 @@
     "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
   },
   "optionErrorCorrectionDescr": {
-    "message": "If a QR code is damaged it can still be read by machines. This value indicates how much of the code can be missing. For more information see <a href='https://en.wikipedia.org/wiki/QR_code#Error_correction'>this Wikipedia article</a>.",
-    "description": "The description of the error correction option."
+    "message": "!HTML! If a QR code is damaged it can still be scanned. This value indicates how much of the code can be missing. <br> For more information see $URL$.",
```

Comment:
Also this `!HTML!` "trigger string" (or however one wants to call this) needs soem documentation in the Contributing.md.